scipy/_build_utils/_fortran.py

```diff
@@ -20,7 +20,7 @@ def uses_mkl(info):
 
 def needs_g77_abi_wrapper(info):
     """Returns True if g77 ABI wrapper must be used."""
-    return uses_mkl(info)
+    return str(os.getenv("SCIPY_USE_G77_ABI_WRAPPER", uses_mkl(info))) == "True"
```

Comment:
That's a bit too fussy parsing, elsewhere we do
```
try:
    needs_wrapper = int(os.environ.get("SCIPY_USE_G77_ABI_WRAPPER", "0"))
except ValueError:
    needs_wrapper = False
...
```

Reply:
Did you mean something like,
```python

    try:
        needs_wrapper = int(os.environ["SCIPY_USE_G77_ABI_WRAPPER"]) != 0
    except KeyError:
        needs_wrapper = uses_mkl(info)
```

Reply:
Yes, for example. 
Also, in case we don't have a place in the build (reference?) documentation mentioning environment variables, it could be added.